packages/app/src/constants.js

```diff
@@ -29,3 +29,16 @@ export const OVERLAY_STATE = {
   ABOUT: 'about',
   KEYBINDINGS: 'keybindings'
 }
+
+export const FALLBACK_SLUGS = {
```

Comment:
@breadadams let's drop this, I prefer to write a network rewrite rather than keep legacy code